.github/workflows/int-and-connected-test-run.yml

```diff
@@ -159,137 +91,76 @@ jobs:
         with:
           actions_subcommand: 'helmdeploy'
           helm_create_secret_manager_secret_version: 0.0.8
-          helm_datarepo_api_chart_version: 0.0.739
-          helm_datarepo_ui_chart_version: 0.0.360
+          helm_datarepo_api_chart_version: 0.0.744
+          helm_datarepo_ui_chart_version: 0.0.362
           helm_gcloud_sqlproxy_chart_version: 0.19.13
           helm_oidc_proxy_chart_version: 0.0.44
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Fetch gitHash for deployed integration version"
+      - name: Fetch gitHash for deployed integration version
         id: configuration
         run: |
           git_hash=$(git rev-parse --short HEAD)
           echo "git_hash=${git_hash}" >> $GITHUB_OUTPUT
           echo "Latest git hash for this branch: $git_hash"
-      - name: "Wait for deployment to come back online"
+      - name: Wait for deployment to come back online
         uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@0.74.0
         timeout-minutes: 20
         env:
           DESIRED_GITHASH: ${{ steps.configuration.outputs.git_hash }}
           DEPLOYMENT_TYPE: 'api'
-      - name: "Run Test Runner smoke tests via Gradle"
+      - name: Run test runner smoke tests via Gradle
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'gradletestrunnersmoketest'
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Run integration tests via Gradle"
+      - name: Run integration tests via Gradle
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
+        env:
+          AZURE_CREDENTIALS_APPLICATIONID: ${{ env.AZURE_CREDENTIALS_APPLICATIONID }}
+          AZURE_CREDENTIALS_HOMETENANTID: ${{ env.AZURE_CREDENTIALS_HOMETENANTID }}
         with:
           actions_subcommand: 'gradleinttest'
           pgport: ${{ job.services.postgres.ports[5432] }}
           test_to_run: 'testIntegration'
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-        env:
-          AZURE_CREDENTIALS_APPLICATIONID: ${{ env.AZURE_CREDENTIALS_APPLICATIONID }}
-          AZURE_CREDENTIALS_HOMETENANTID: ${{ env.AZURE_CREDENTIALS_HOMETENANTID }}
-      - name: "Clean state lock from used Namespace on API deploy"
+      - name: Clean state lock from used Namespace on API deploy
         if: always()
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'k8_checknamespace_clean'
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-      - name: "Clean whitelisted Runner IP"
+      - name: Clean whitelisted Runner IP
         if: always()
         uses: broadinstitute/datarepo-actions/actions/main@0.74.0
         with:
           actions_subcommand: 'gcp_whitelist_clean'
           sa_b64_credentials: ${{ secrets.SA_B64_CREDENTIALS }}
-  report-to-sherlock:
```

Comment:
Reviewer note: do we still want to report any of this information?

Reply:
How would sherlock know about a branch version if this was removed? Is there something else that's registering branch builds with sherlock? If anything, it'd be nice to enable publishing to sherlock without `needs: deploy_test_integration` as it'd b nice to use a branch build in a bee without having to wait for integration tests to pass first.

Reply:
It's not clear what the `sherlock` functionality does. Is this something we need?

Reply:
I restored this functionality, but I'm not sure it's necessary.

Reply:
We do want to report versions to Sherlock: doing so is what makes Beehive aware of our feature branches, so that devs can deploy them to their Bees.  If a comment or additional description would help make that clear (so that this step isn't inadvertently removed in the future), I'm all for adding them.

> If anything, it'd be nice to enable publishing to sherlock without needs: deploy_test_integration as it'd b nice to use a branch build in a bee without having to wait for integration tests to pass first.

@pshapiro4broad I agree with you, that would be nice. I'm not sure it's as simple as just removing the `needs: deploy_test_integration` so leaving it in for now is my preference.  See related ticket: https://broadworkbench.atlassian.net/browse/DCJ-38